python/doc/theory/reliability_sensitivity/form_approximation.rst

```diff
@@ -3,106 +3,75 @@
-     diffeomorphism from :math:`\supp{\vect{X}}` into :math:`\Rset^n`,
-     such that the distribution of the random vector
-     :math:`\vect{U}=T(\vect{X})` has the following properties :
-     :math:`\vect{U}` and :math:`\mat{R}\,\vect{U}` have the same
-     distribution for all rotations :math:`\mat{R}\in{\cS\cO}_n(\Rset)`.
-   | The usual isoprobabilistic transformations are the Generalized
-     Nataf transformation and the Rosenblatt one.
-   | The mapping of the limit state function is
-     :math:`G(\vect{U}\,,\,\vect{d}) =  g(T^{-1}(\vect{U})\,,\,\vect{d})`.
-     Then, the event probability :math:`P_f` rewrites :
-
-     .. math::
-       :label: PfU3
-
-          P_f = \Prob{G(\vect{U}\,,\,\vect{d})\leq 0} = \int_{\Rset^n} \boldsymbol{1}_{G(\vect{u}\,,\,\vect{d}) \leq 0}\,f_{\vect{U}}(\vect{u})\,d\vect{u}
-
-     where :math:`f_{\vect{U}}` is the density function of the
-     distribution in the standard space : that distribution is spherical
-     (invariant by rotation by definition). That property implies that
-     :math:`f_{\vect{U}}` is a function of :math:`||\vect{U}||^2` only.
-     Furthermore, we suppose that outside the sphere which tangents the
-     limit state surface in the standard space, :math:`f_{\vect{U}}` is
-     decreasing.
-
-#. | Find the design point :math:`\vect{u}^*` which is the point verifying
-     the event of maximum likelihood : the decreasing hypothesis of the
-     standard distribution :math:`f_{\vect{U}}` outside the sphere which
-     tangents the limit state surface in the standard space implies that
-     the design point is the point on the limit state boundary the nearest
-     to the origin of the standard space. Thus, :math:`\vect{u}^*` is the
-     result of a constrained optimization problem.
-
-#. | In the standard space, approximate the limit state surface in the
-     standard space by a linear surface at the design point
-     :math:`\vect{u}^*`. Then, the probability :math:`P_f` (:eq:`PfU3`) where
-     the limit state surface has been approximated by a linear surface
-     (hyperplane) can be obtained exactly, thanks to the rotation
-     invariance of the standard distribution :math:`f_{\vect{U}}` :
-
-   .. math::
-     :label: PfFORM
+
+The objective of FORM is to evaluate :math:`P_f`. The method proceeds in three steps:
+
+**Step 1**: Map the probabilistic model in terms of :math:`\vect{X}` thanks to an
+isoprobabilistic transformation :math:`T` which is a diffeomorphism from
+:math:`\supp{\vect{X}}` into :math:`\Rset^n`, such that the distribution of the random vector
+:math:`\vect{U}=T(\vect{X})` has
+the following properties: :math:`\vect{U}` and :math:`\mat{R}\,\vect{U}` have the same distribution for all rotations :math:`\mat{R}\in{\cS\cO}_n(\Rset)`.
+The usual isoprobabilistic transformations are the Generalized
+Nataf transformation and the Rosenblatt one.
+The mapping of the limit state function is
+:math:`G(\vect{U}\,,\,\vect{d}) =  g(T^{-1}(\vect{U})\,,\,\vect{d})`.
```

Comment:
I would suggest the same notations as suggested in python/doc/theory/numerical_methods/isoprobabilistic_transformation.rst, since there is a potential confusion between upper case G and lower case g.
```suggestion
:math:`h(\vect{U}\,,\,\vect{d}) =  g(T^{-1}(\vect{U})\,,\,\vect{d})`.
```

Reply:
I think it is not a good idea because of the amount of the doc to be changed....

Reply:
I agree that this requires some work. Compared to the work already done, however, this is not that significant. Keeping a poor notation choice cannot be a solution neither. Secondly, that notation is not used in many places: at most 10 pages. Finally, new helps pages such as the PR [2272](https://github.com/openturns/openturns/pull/2272) propagate that poor notation choice for consistency reasons. Hence, not fixing this in this page means to fix more pages in a near future. While we are at fixing these particular pages (and not others), why not finish that?

To quantify the required changes, I used the next bash command:
```
$ find python/doc/theory -name "*.rst" -print | xargs -d "\n" grep -w "G"
```
and I find 23 matching lines, half of which are bibliographic references. After manual filtering, here are the 14 results:
```
python/doc/theory/reliability_sensitivity/sorm_approximation.rst:    P_f = \Prob{G(\vect{U}\,,\,\vect{d})\leq 0} = \int_{\Rset^n} \boldsymbol{1}_{G(\vect{u}\,,\,\vect{d}) \leq 0}\,f_{\vect{U}}(\vect{u})\,d\vect{u}
python/doc/theory/reliability_sensitivity/sensitivity_form.rst:  :math:`G(\vect{U}\,,\,\vect{\theta}) =  g[T^{-1}(\vect{U}\,,\,\vect{\theta}), \vect{d}]`,
python/doc/theory/reliability_sensitivity/sensitivity_form.rst:      \nabla_{\vect{\theta}} \beta_{HL} =  \displaystyle +\frac{1}{||\nabla_{\vect{\theta}} G(U_{\vect{\theta}}^{*}, \vect{d})||} \nabla_{\vect{u}} G(U_{\vect{\theta}}^{*}, \vect{d}).
python/doc/theory/reliability_sensitivity/subset_sampling.rst:These thresholds are estimated as :math:`\alpha_{SS}` quantiles from the set of :math:`N` samples of LSF outputs :math:`\mathcal{G}_{u,s} = \{ \overset{\circ}{g}(U^{(j)}) \}_{j=1}^N` with :math:`\alpha_{SS} \in ]0, 1[` the *rarity parameter*. Consequently, one can notice that
python/doc/theory/reliability_sensitivity/form_approximation.rst::math:`G(\vect{U}\,,\,\vect{d}) =  g(T^{-1}(\vect{U})\,,\,\vect{d})`.
python/doc/theory/reliability_sensitivity/form_approximation.rst:    P_f = \Prob{G(\vect{U}\,,\,\vect{d})\leq 0} = \int_{\Rset^n} \boldsymbol{1}_{G(\vect{u}\,,\,\vect{d}\leq 0}\,f_{\vect{U}}(\vect{u})\,d\vect{u}
python/doc/theory/reliability_sensitivity/strong_maximum_test.rst:the event has the new expression defined from the transformed limit state function of the model :math:`G`: :math:`\cD_f = \{\vect{U} \in \Rset^n \, / \, G(\vect{U}\,,\,\vect{d}) \le 0\}`.
python/doc/theory/numerical_methods/isoprobabilistic_transformation.rst:    P_f = \Prob{G(\vect{U}\,,\,\vect{d})\leq 0} = \int_{\Rset^n} \boldsymbol{1}_{G(\vect{u}\,,\,\vect{d}) \leq 0}\,f_{\vect{U}}(\vect{u})\,d\vect{u}
python/doc/theory/numerical_methods/isoprobabilistic_transformation.rst::math:`G=g\circ T^{-1}`.
python/doc/theory/probabilistic_modeling/estimate_spectral_density_function.rst:     \hat{G}_{\vect{x}}(f_p,T)&=&\frac{2}{T}\vect{X}_{k}(f_p,T)\,{\vect{X}_{k}(f_p,T)^*}^t,\quad p=0,\dots,L/2-1\end{aligned}
python/doc/theory/probabilistic_modeling/estimate_spectral_density_function.rst:-  Bias\ :math:`=\mathbb{E}[\hat{G}_{\vect{x}}(f_p, T)-G_{\vect{X}}(f_p)]=(\frac{1}{T}W_B(f_p, T)-\delta_0)*G_{\vect{X}}(f_p)`
python/doc/theory/probabilistic_modeling/estimate_spectral_density_function.rst:.. math:: \hat{G}_{\vect{x}}(f_p,T)=\frac{2}{KT}\sum_{k=0}^{K-1}\vect{X}^{(k)}(f_p,T)\vect{X}^{(k)}(f_p,T)^t
python/doc/theory/probabilistic_modeling/estimate_spectral_density_function.rst:.. math:: \mathbb{E}[\hat{G}_{\vect{x}}(f_p, T)-G_{\vect{X}}(f_p)=(\frac{1}{T}W(f_p, T)-\delta_0)*G_{\vect{X}}(f_p)
```

Not that big of a deal after all, isn't it?

Reply:
ok... It is really because it is you... :-)
In the estimate_spectral_density_function.rst file, G is the spectral density funciton : so I did not change it!